docs/architecture/grpc-for-wcf-developers/wcf-bindings.md

```diff
@@ -1,11 +1,35 @@
 ---
 title: WCF bindings and transports - gRPC for WCF Developers
-description: TO BE WRITTEN
+description: How the various WCF bindings and transports compare to gRPC
 author: markrendle
 ms.date: 09/02/2019
 ---
 
 # WCF bindings and transports
 
+WCF has lots of different built in *bindings* specifying different network protocols, wire formats and other implementation details. gRPC effectively has just one network protocol and one wire format. However, you are still likely to discover that gRPC offers the best solution in most cases. What follows is a short discussion about the most relevant WCF Bindings and how they compare to their equivalent in gRPC.
```

Comment:
I thought about mentioning the alternative wire formats but I think it would overcomplicate the guide in the context of its intended purpose and readership. It will be easy to add a new section on custom transports when 5.0 launches.

Reply:
> My fear is that is content won't age well

Updating the doc in the future won't be that difficult...